controllers/usersignup/masteruserrecord.go

```diff
@@ -56,6 +56,10 @@ func migrateOrFixMurIfNecessary(mur *toolchainv1alpha1.MasterUserRecord, nstempl
 		mur.Annotations[toolchainv1alpha1.MasterUserRecordEmailAnnotationKey] = userSignup.Annotations[toolchainv1alpha1.UserSignupUserEmailAnnotationKey]
 		changed = true
 	}
+	// TODO: remove this after UserAccount.NStemplateSet has been removed (CRT-1321)
+	if len(mur.Spec.UserAccounts) > 0 && mur.Spec.UserAccounts[0].Spec.NSTemplateSet != nil {
+		mur.Spec.TierName = mur.Spec.UserAccounts[0].Spec.NSTemplateSet.TierName
```

Comment:
you should have also modified the `changed` variable so it updates the MUR, right?
```suggestion
		mur.Spec.TierName = mur.Spec.UserAccounts[0].Spec.NSTemplateSet.TierName
		changed = true
```

Reply:
I'm sorry, I don't understand your comment vs the change you're suggesting :/

Reply:
never mind, I applied the change

Reply:
You needed to set the `changed` variable to `true` - as is proposed in the suggestion, otherwise, the controller wouldn't update the MasterUserRecord resource. 
Sorry if I wasn't clear enough - I wrote it very quickly.